src/BitcoinHelpers.js

```diff
@@ -476,7 +506,10 @@ const BitcoinHelpers = {
       // Combine witness
       let signedTransaction
       try {
-        signedTransaction = bcoin.TX.fromRaw(unsignedTransaction, "hex").clone()
+        signedTransaction = bcoin.TX.fromRaw(
+          Buffer.from(unsignedTransaction, "hex"),
```

Comment:
We got lucky here that bcoin could take a hex string in `fromRaw` despite declaring that it always takes a `Buffer`. This does the buffer conversion on our end to conform with the declared API.